index.bs

```diff
@@ -273,21 +273,21 @@ For the main <dfn>sanitize</dfn> operation, using a {{ParentNode}} |node|, a
            this algorithm will be used in different contexts, this assumption
            needs to be re-examined.
   1. If |child| [=implements=] {{Text}}:
-    1. [=continue=].
+    1. Then [=continue=].
```

Comment:
Can we inline these, I think that would be much clearer?

> 1. If child implements Text, then continue.
> 2. Otherwise, if child implements Comment:

(Also, specifications use "Otherwise", not "Else".

Reply:
Done. (Here and elsewhere.)